changelog/4861.bugfix.rst

```diff
@@ -0,0 +1 @@
+Improve validation of contents written to captured output so it behaves the same as when capture is disabled.
```

Comment:
@blueyed I tweaked the CHANGELOG a bit, hope you don't mind.

Reply:
@nicoddemus 
Great!